python/hidet/graph/nn/linear.py

```diff
@@ -61,3 +62,23 @@ def forward(self, x: Tensor) -> Tensor:
         if self.bias is not None:
             x = ops.add(x, self.bias)
         return x
+
+
+class SymQuantLinearTransposed(Module):
+    def __init__(self, weight: Tensor, bias: Optional[Tensor] = None, quant_type: str = 'int8'):
+        super().__init__()
+        self.in_features = weight.shape[0]
+        self.out_features = weight.shape[1]
+        qweight, scale = ops.symmetric_quantize(weight, quant_type=quant_type, dims=[-1])
+        self.qweight = qweight
+        self.scale = scale
+        self.bias = bias
```

Comment:
This operator looks very different from the normal operator (e.g., only pass the operator config to the __init__ and create the tensor inside __init__). Is this nn module necessary? I can hardly think of a way to use it in hidet because the quantized operator is usually added in a pass.